docs/guides/function-template-hamiltonian-simulation.ipynb

```diff
@@ -16,34 +16,28 @@
      "version-info"
     ]
    },
-   "source": [
-    "<details>\n",
-    "<summary><b>Package versions</b></summary>\n",
-    "\n",
-    "The code on this page was developed using the following requirements.\n",
-    "We recommend using these versions or newer.\n",
-    "\n",
-    "```\n",
-    "qiskit[all]~=1.2.4\n",
-    "qiskit-ibm-runtime~=0.32.0\n",
-    "qiskit-serverless~=0.18.0\n",
-    "qiskit-ibm-catalog~=0.2.0\n",
-    "```\n",
-    "</details>"
-   ]
+   "source": []
   },
   {
    "cell_type": "markdown",
-   "id": "5103c621-809d-48e9-ac86-2bf02916e2af",
+   "id": "b51e81bf-0bbf-4f64-af1e-87fcb443d997",
    "metadata": {},
    "source": [
-    "By now you've seen a basic example of how to get started writing, uploading, and running a program with Qiskit Serverless. If you haven't, start with [What is Qiskit Serverless?](/docs/guides/serverless) for some background information first.\n",
+    "This template encapsulates a workflow to simulate the time evolution of an initial state against a user defined spin-based Hamiltonian and returns a set of specified expectation values using the [AQC addon](./qiskit-addons-aqc). A demonstration of deploying and running this template can be found as a Jupyter notebook in the [project's directory](https://github.com/qiskit-community/qiskit-function-templates/blob/main/physics/hamiltonian_simulation/deploy_and_run.ipynb).\n",
     "\n",
-    "The workloads you are building for your own use cases are likely not as simple as the examples previously shown. For example, you might need to carefully consider what domain-level inputs and outputs are relevant for your application, how to make sure your program can be reusable across a range of those inputs, and what kind of information you need returned during the execution of the job so you can better evaluate the progress of your workload. You might even want to incorporate a \"dry run\" mode so you can test the impact of a set of particular inputs before sending the resulting circuits to hardware.\n",
+    "Similar to the chemistry workflow it is also divided into several sections:\n",
     "\n",
-    "A function template is an example of a realistic workload that uses a specific application domain to contextualize these aspects. It is meant to be a starting point for you to modify for your own needs so you don't have to start from scratch.\n",
+    "1. Input validation\n",
```

Comment:
I think you can combine input validation with mapping like you have on the Chemistry guide: "1. Collecting input and mapping the problem" and then add a step two for transpiling to ISA, followed by a step 3 for execution